MAPL_Base/MAPL_IO.F90

```diff
@@ -239,11 +239,12 @@ module MAPL_IOMod
 
   contains
 
-      subroutine ArrDescrInit(ArrDes,comm,im_world,jm_world,nx,ny,num_readers,num_writers,is,ie,js,je,rc)
+      subroutine ArrDescrInit(ArrDes,comm,im_world,jm_world,lm_world,nx,ny,num_readers,num_writers,is,ie,js,je,rc)
```

Comment:
It would be better if lm_world could be an optional argument so that it does not break backward compatibility.  Unfortunately, rc is already optional, so doing this the "right way" (TM) would still force existing clients to add "rc=...".

Reply:
ok